setup.cfg

```diff
@@ -16,4 +16,4 @@ line_length = 88
 multi_line_output = 3
 include_trailing_comma = True
 skip = ve/,env/
-known_third_party = boto3,celery,dj_database_url,django,environ,freezegun,hashids,json2html,kombu,moto,phonenumber_field,pkg_resources,prometheus_client,pytest,raven,recommonmark,requests,responses,rest_framework,sentry_sdk,setuptools,six,temba_client
+known_third_party = celery,dj_database_url,django,environ,freezegun,hashids,json2html,jsonschema,kombu,phonenumber_field,phonenumbers,pkg_resources,prometheus_client,pytest,pytz,raven,recommonmark,redis,requests,responses,rest_framework,sentry_sdk,setuptools,six,temba_client
```

Comment:
@MatthewWeppenaar did you update these or did an automated action update them?
I'm concerned about the removal of some of the libraries (especially boto3) so just want to check if something prompted you to do it or if it's automated.

Reply:
ok no that's fine. Just wanted to understand the reason behind it but if it was an automated process then it was probably removed in a previous change and the setup.cfg file wasn't updated.